src/shogun/lib/config.h.in

```diff
@@ -103,6 +103,8 @@
 #cmakedefine USE_MAP_BACKEND 1
 #cmakedefine USE_SORTED_VECTOR_BACKEND 1
 
+#cmakedefine SPDLOG_FMT_EXTERNAL 1
```

Comment:
MacOS does not bundle fmt with spdlog anymore. Instead fmt is a dependency and installed separately. So spdlog needs to be told to use external fmt instead of the missing bundled one.
This variable is set automatically when finding spdlog in config mode.